content/realtime/channels.textile

```diff
@@ -255,11 +372,15 @@ channel.publish("action", data: "boom!")
 
 Normally, errors in attaching to a channel are communicated through the attach callback. For implicit attaches (and other cases where a channel is attached or reattached automatically, e.g. following the library reconnecting after a period in the @suspended@ state), there is no callback, so if you want to know what happens, you'll need to listen for channel state changes.
 
+h3(#batch-publish). Batch publishing
+
+It is common for a single message to be intended for multiple channels. With a realtime connection, you can effectively send a message to multiple channels at once by allowing multiple concurrent publish operations. If you wish to send a message to multiple channels within a single operation, you can make use of the "REST batch API":/rest-api/beta#batch.
```

Comment:
Since realtime doesn't support batch publishing I do think having this heading is misleading. I realise you wanted to keep the headings consistent, but the APIs are different, and I don't think it's appropriate in this case.

Perhaps this section should literally be called "Publishing to multiple channels".

The the text could be:

Often it is necessary to publish a single message in multiple channels at the same time. In the realtime API, this is achieved simply by making multiple separate publish() requests, once in each of the channels in question, and the realtime protocol allows for those concurrent requests to be in-flight simultaneously, so the publish on one channel is not delayed waiting for completion of operations in other channels.

It is also possible to publish one or more messages into multiple channels in a single operation using the "REST batch API":/rest-api/beta#batch.